docs/source/noise/column_noise.rst

```diff
@@ -4,18 +4,145 @@
+It takes one parameter:
+
+.. list-table:: Parameters to the choose_wrong_option noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability that, for a cell in the column being configured, the wrong option is chosen.
+    - 0.01 (1%)
+
+Write the wrong ZIP code digits
+-------------------------------
+
+When reporting a ZIP code on a survey or form, people may misremember or misreport
+the digits.
+They are probably more likely to do this for the last few digits (which identify
+the small, specific area) than the first few (which will be the same over a larger area).
+
+This noise type is called :code:`write_wrong_zipcode_digits` in the configuration.
+It takes two parameters:
+
+.. list-table:: Parameters to the write_wrong_zipcode_digits noise type
+  :widths: 1 5 3
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability of a cell being *considered* to have this noise type.
+      One way to think about this is the probability that a ZIP code is reported by someone who isn't sure of their ZIP code.
+      Whether or not there are actually any errors depends on the next parameter.
+    - 0.01 (1%)
+  * - :code:`digit_probabilities`
+    - A list of five probabilities, one for each digit in a (5-digit) ZIP code.
+      The first value in this list is the probability that the first digit of the ZIP code will be wrong
+      **given that the cell is being considered for this noise type**.
+      The second value in the list is this same probability but about the second digit, and so on.
+    - [0.04, 0.04, 0.20, 0.36, 0.36]
 
 Misreport age
 -------------
 
+When someone reports their age in years, or especially when someone reports the age of someone else such as a family member,
+they may not get the value exactly right.
+When this noise type occurs, the reported age is off by some amount, for example a year or two older or younger than the
+person actually is.
+
+This noise type is called :code:`misreport_age` in the configuration.
+It takes two parameters:
+
+.. list-table:: Parameters to the misreport_age noise type
+  :widths: 1 5 3
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability of each age value being misreported.
+    - 0.01 (1%)
+  * - :code:`possible_age_differences`
+    - One of two options:
+
+        * A list of possible differences to add to the true age to get the misreported age.
+          A negative number means that the reported age is too young, while a positive number means it is too old.
+          Each difference is equally likely.
+        * A dictionary mapping from possible differences to the corresponding probabilities of those differences.
+          This is like the list option except that it allows some differences to be more likely than others.
```

Comment:
```suggestion
          This is like the list option except that it allows some age differences to be more likely than others.
```
It might be a little clearer to be more specific